src/App.tsx

```diff
@@ -9,24 +9,22 @@ interface Todo {
 function App() {
   const [items, setItems] = useState<Todo[]>([]);
 
+  const todos = items.reduce((acc: any[], obj) => {
+    const values = Object.values(obj).slice(1, 3);
+    return [...acc, values];
```

Comment:
Try to simplify this part of the code.

- "Object.values" is not required.
- slice may cut the wrong values if the order of object fields changes. Better not to use it.

Reply:
Done